apps/frontend/src/renderer/components/linear/components/LinearTicketList.tsx

```
@@ -0,0 +1,212 @@
+
+    observer.observe(trigger);
+
+    return () => {
+      observer.disconnect();
+    };
+  }, [handleIntersection, onLoadMore, viewportElement]);
+
+  // Loading state (initial load) - show skeleton loaders
+  if (isLoading && tickets.length === 0) {
+    return (
+      <div className="flex-1 overflow-y-auto" role="status" aria-live="polite" aria-busy="true" aria-label="Loading tickets">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)